src/crud/representation.py

```
@@ -4,11 +4,60 @@
+        .where(models.Peer.name == peer_name)
+        .values(
+            internal_metadata=models.Peer.internal_metadata.op("||")(
+                {"peer_card": peer_card}
+            )
+        )
+    )
+    result = await db.execute(stmt)
+    if result.rowcount == 0:
+        raise exceptions.ResourceNotFoundException(
+            f"Peer {peer_name} not found in workspace {workspace_name}"
+        )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)